shared/sdl/sdl_window.cpp

```diff
@@ -723,6 +729,7 @@ window_t WIN_Init( const windowDesc_t *windowDesc, glconfig_t *glConfig )
 	Cmd_AddCommand("minimize", GLimp_Minimize);
 
 	r_sdlDriver			= Cvar_Get( "r_sdlDriver",			"",			CVAR_ROM );
+	r_allowsoftwaregl	= Cvar_Get( "r_allowsoftwaregl",	"0",		CVAR_ARCHIVE|CVAR_LATCH );
```

Comment:
Can you rename the cvar and variable to use camelCase? e.g. `r_allowSoftwareGL` Just to be consistent with everything else.